specs/simple-serialize.md

```diff
@@ -116,26 +110,17 @@ return value
 |  `hash32` | Hash size of ``keccak`` or `blake2b[0.. < 32]`. |
 |  `hash96` | BLS Public Key Size.                            |
 |  `hash97` | BLS Public Key Size with recovery bit.          |
+|  `hashN`  | Hash of arbitrary byte length, N                |
```

Comment:
It's a little confusing if we keep `hash32`, `hash96`, `hash97`, and `hashN` in the table when they can all be replaced with `hashN`. I think we can just remove the description of `hash32`, `hash96`, and `hash97`.

Reply:
nitpick: add a period in the end and markdown code mark:

Hash of arbitrary byte length `N`.

Reply:
Since `HashN` is a custom type like `Address`, IMO we can use CapWords notation `HashN`  in this spec.

Reply:
In the implementation, it is assumed that everything is lowercase. I think we should be clearer and more consistent across the board. Maybe for another pr